execute/executetest/table.go

```diff
@@ -48,9 +48,14 @@ func (t *Table) Normalize() {
 			if len(t.Data) > 0 {
 				t.KeyValues[j] = t.Data[0][idx]
 			}
-			v := values.New(t.KeyValues[j])
-			if v.Type() == semantic.Invalid {
-				panic(fmt.Errorf("invalid value: %s", t.KeyValues[j]))
+			var v values.Value
+			if t.KeyValues[j] == nil {
+				v = values.NewNull(flux.SemanticType(t.ColMeta[idx].Type))
```

Comment:
Is there a way to make it so `values.New(nil)` works  instead of adding a new function?

Reply:
I thought of that too, but a values.Value has to have a type.

Reply:
We want `null` values to have a type, so I don't see a way to do it just by passing `nil`.  If `New` accepted a pointer to a value, then it would be possible by passing a null pointer... but that seems kind of hacky to me.